libcob/common.c

```diff
@@ -1975,16 +1975,12 @@ cob_cmp_all (cob_field *f1, cob_field *f2)
 	return ret;
 }
 
-/* compare content of field 'f1' to content of 'f2', space padded,
-   using the optional collating sequence of the program */
```

Comment:
let's include the old doc comment, adjusted to "strings, space padded, using an optional collating sequence" here so this "local exported" function has a doc comment